python/legate_dataframe/testing.py

```diff
@@ -18,7 +14,17 @@
 from legate_dataframe import LogicalColumn, LogicalTable
 
 
-def as_cudf_dataframe(obj: Any, default_column_name: str = "data") -> cudf.DataFrame:
+def try_import_cudf():
+    # cudf can exist but not be usable if the GPU driver is not available
+    cudf = None
+    try:
+        cudf = pytest.importorskip("cudf")
```

Comment:
I suppose before long we might have to run some cudf function to be sure it is functional (i.e. cudf might get smart enough to import successfully without a GPU, although maybe that is only relevant for `pylibcudf` and won't happen).